src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/RecallAuditComponentTest.kt

```diff
@@ -53,7 +56,108 @@ class RecallAuditComponentTest : ComponentTestBase() {
   }
 
   @Test
-  fun `get audit summary for a recall that has been booked`() {
+  fun `get lastReleaseDate (LocalDate value) audit history`() {
+    val savedRecall = authenticatedClient.bookRecall(BookRecallRequest(nomsNumber, FirstName("Brian"), null, LastName("Badgering")))
+    val recallId = savedRecall.recallId
+
+    val updatedRecall = authenticatedClient.updateRecall(recallId, UpdateRecallRequest(lastReleaseDate = LocalDate.now()))
```

Comment:
Would be interesting to add some parameterized test examples - though also I haven't tried for you.

Reply:
Done